### ISSUE TITLE
Need test cases.

### ISSUE THREAD

**Issue by lsetiawan** [Freshwater-Initiative/Observatory#3]
Probably okay for now, but it'd be great to have a couple of test cases to test the functions.

**Comment by jphuong** [comment]
I've tried pip install ogh, but I'm not able to import the library. There seems to be an encryption error going on.

![screen shot 2018-03-14 at 2 11 38 pm](https://user-images.githubusercontent.com/16712523/37431213-daa99dd6-2791-11e8-9ef4-faa38d2aeb2a.png)

**Comment by lsetiawan** [comment]
@jphuong How did you install `ogh`? Just pip install?

**Comment by jphuong** [comment]
yes. "pip install ogh". I looked up this error and found this related thread. [https://github.com/Miserlou/Zappa/issues/854] They suggest that it might have something to do with the .pyc, something strange about the python interpreter that could differ between the python 2.7 and python 3.6 versions.

**Comment by lsetiawan** [comment]
There are many different issues that can be happening, probably related more dependencies, etc. Try installing the dependencies first with conda then do pip install. That works for me.

**Comment by lsetiawan** [comment]
Gotta figure out what's missing. I can't even complete pip install. What python version are you using?

**Comment by jphuong** [comment]
Python environment:
3.6.4.final.0

conda version:
4.3.29

It seems that you, me, and Christina are each getting different error messages. Christina's error message has to do with line 14 in the ogh.py. After inspecting that error code, I'm just going remove that line as it was a constraint from a different environment that no longer applies. That doesn't resolve your issue, Don.

**Comment by lsetiawan** [comment]
It's in conda now. Try installing from conda. `conda install -c conda-forge ogh`

**Comment by lsetiawan** [comment]
It's in conda. I think the error is just in dependencies. I am having no problem now as long as I have installed all of the dependencies first. Of course installing via conda is the easiest way out. https://anaconda.org/conda-forge/ogh
